README.md

```diff
@@ -36,6 +36,15 @@ function App(props) {
 }
 ```
 
+## Development
+
+To get the development setup make sure to have [NodeJS](https://nodejs.org/en/download/) installed.
+As soon as you are set up, clone the project and execute
+
+```
+npm install
+npm run all
+```
```

Comment:
I would also add a quick note on the example, as it is not obvious that you need to open a sub-directory:
```
## Example

To run the example, execute `npm run start:example` and open http://localhost:5000/public in your Webbrowser.
```

Reply:
Thanks! I added this via https://github.com/bpmn-io/react-bpmn/pull/15/commits/2161ee5933dc12f5502baf69dafe4410d517da54